internal/pkg/version/check_remote.go

```diff
@@ -22,7 +22,7 @@ type checker struct {
 	logger *zap.SugaredLogger
 }
 
-func NewChecker(parentCtx context.Context, logger *zap.SugaredLogger, envs *env.Map) *checker {
+func NewGitHubChecker(parentCtx context.Context, logger *zap.SugaredLogger, envs *env.Map) *checker {
```

Comment:
Vo `version` pkg je aj kontrola aktualneho release na GitHube, .. tak som to nazval trocha lepsie `version.NewGitHubChecker`.